lightning/src/util/events.rs

```diff
@@ -515,6 +540,11 @@ impl Writeable for Event {
 					(2, payment_hash, required),
 				})
 			},
+			&Event::OpenChannelRequest { .. } => {
+				16u8.write(writer)?;
```

Comment:
This needs to be odd - our read behavior is "even, and we dont understand it -> assume we cant read anything and it is an important event, refuse to deserialize at all", but here its an unimportant (so unimportant we *want* to throw it away) event, so we should use an odd value to indicate that. We should also add a comment on the read-side code noting that the value we use is used for `OpenChannelRequest` (maybe a match branch).

Reply:
Thanks again for the detailed input! I'll assign it to 17 instead.